libs/sdk-ui-ext/src/internal/components/configurationControls/DropdownControl.tsx

```diff
@@ -33,11 +34,19 @@ class DropdownControl extends React.PureComponent<IDropdownControlProps & Wrappe
         disabled: false,
         width: 117,
         showDisabledMessage: false,
+        pushInitialValue: false,
     };
 
     constructor(props: IDropdownControlProps & WrappedComponentProps) {
         super(props);
         this.onSelect = this.onSelect.bind(this);
+        this.getSelectedItem = this.getSelectedItem.bind(this);
+
+        const { value, pushInitialValue } = props;
+
+        if (pushInitialValue) {
```

Comment:
This should be in `componentDidMount`, side effects in constructor are not desireable.